infrastructure/index.ts

```diff
@@ -50,43 +52,28 @@ const contentBucket = new aws.s3.Bucket(
     },
 );
 
-// contentBucket needs to have the "public-read" ACL so its contents can be ready by CloudFront and
-// served. But we deny the s3:ListBucket permission to prevent unintended disclosure of the bucket's
-// contents.
-const denyListPolicyState: aws.s3.BucketPolicyArgs = {
-    bucket: contentBucket.bucket,
-    policy: contentBucket.arn.apply((arn: string) => JSON.stringify({
+// The website bucket needs to have the "public-read" ACL so their contents can be read by
```

Comment:
Super nitpick: "its" still reads better to me, but happy to go with "their" if it is more correct.

```suggestion
// The website bucket needs to have the "public-read" ACL so its contents can be read by
```

Reply:
Ah yes, you're right -- this was written when there were two buckets to manage, but now we only have one.